vms/platformvm/network/network.go

```diff
@@ -142,12 +211,13 @@ func (n *network) IssueTx(ctx context.Context, tx *txs.Tx) error {
 	}
 
 	txID := tx.ID()
-	n.gossipTx(ctx, txID, msgBytes)
-	return nil
+	n.legacyGossipTx(ctx, txID, msgBytes)
```

Comment:
It's interesting to note that unit tests previously passed because we suppress the error here... even though a failed send is considered fatal